src/filter-image/FilterImage.tsx

```diff
@@ -0,0 +1,61 @@
+import { ImageProps, View, Image as RNImage, StyleSheet } from 'react-native';
+import {
+  Svg,
+  FeColorMatrix,
+  Filter,
+  Image,
+  FeColorMatrixProps,
+} from '../index';
+import { FilterPrimitiveStandardAttributes } from '../elements/filters/FilterPrimitive';
+
+export interface FilterImageProps extends ImageProps {
+  filters: Array<
+    ({ name: 'colorMatrix' } & FeColorMatrixProps) &
+      FilterPrimitiveStandardAttributes
+  >;
+}
+
+const getFilters = (filters: FilterImageProps['filters']) => {
+  return filters?.map((filter, index) => {
+    const { name, ...filterProps } = filter;
+    switch (name) {
+      case 'colorMatrix':
+        return <FeColorMatrix key={`${filter}-${index}`} {...filterProps} />;
+      default:
+        return null;
+    }
+  });
+};
+
+export const FilterImage = (props: FilterImageProps) => {
+  const { source, style, ...imageProps } = props;
+
+  const src = RNImage.resolveAssetSource(source);
+  const styles = StyleSheet.flatten(style);
+  const width = props.width || styles?.width || src.width;
+  const height = props.height || styles?.height || src.height;
+
+  return (
+    <View
+      style={[
+        styles,
+        {
+          width,
+          height,
+          overflow: 'hidden',
```

Comment:
Why do we use this option here?

Reply:
This is a hack to force the svg component to behave like the Image component, i.e. respect `borderRadius`